samples/charts/data-chart/series-marker-template/src/DataChartSeriesMarkerTemplate.tsx

```diff
@@ -0,0 +1,97 @@
+// data chart's elements for column series:
+import { IgrNumericYAxis } from 'igniteui-react-charts';
+import { IgrCategoryXAxis } from 'igniteui-react-charts';
+import { IgrColumnSeries } from 'igniteui-react-charts';
+// data chart's modules:
+import { IgrDataChart } from 'igniteui-react-charts';
+import { IgrDataChartCoreModule } from 'igniteui-react-charts';
+import { IgrDataChartCategoryModule } from 'igniteui-react-charts';
+import { IgrDataChartInteractivityModule } from 'igniteui-react-charts';
+import { DataTemplateMeasureInfo } from 'igniteui-react-core';
+import { DataTemplateRenderInfo } from 'igniteui-react-core';
+import * as React from 'react';
+
+IgrDataChartCoreModule.register();
+IgrDataChartCategoryModule.register();
+IgrDataChartInteractivityModule.register();
+
+export default class DataChartSeriesMarkerTemplate extends React.Component<any, any> {
+    public data: any[];
+
+    constructor(props: any) {
+        super(props);
+
+        this.state = { markersType: "Circle" }
+        this.initData();
+    }
+
+    public render(): JSX.Element {
+        return (
+            <div className="igContainer">
+                <div className="igComponent" style={{height: "calc(100% - 35px)"}} >
+                    <IgrDataChart
+                        width="100%"
+                        height="100%"
+                        dataSource={this.data}
+                        isHorizontalZoomEnabled={true}
+                        isVerticalZoomEnabled={true} >
+
+                        <IgrCategoryXAxis name="xAxis" label="Label" interval={1}/>
+                        <IgrNumericYAxis name="yAxis" minimumValue={20} maximumValue={120} />
+
+                        <IgrColumnSeries name="series3"
+                                       valueMemberPath="Value"
+                                       xAxisName="xAxis"
+                                       yAxisName="yAxis"                                       
+                                       markerTemplate={this.getMarker()}
+                                        />
+                   </IgrDataChart>
+                </div>
+            </div>
+        );
+    }
+
+    public initData() {
+
+        let numbers : number[] = [80, 50, 75, 100, 80, 40, 80, 75, 80, 85, 50, 85, 50, 75, 100, 80, 40, 80, 75, 80, 85, 50, 85, 50, 75, 100];
+
+        this.data = [];
+
+        for(let i=0; i<numbers.length; i++){
+            this.data.push({ Label: i.toString(), Value: numbers[i]});
+        }
+    }
+
+    public getMarker(): any
+    {           
+        const markerHeight = 25;
+        const markerWidth = 40;
+
+        return {
+            measure: function (measureInfo: DataTemplateMeasureInfo) {
```

Comment:
missing code for measuring height/width of text that will be displayed in the marker, see [geomap sample](https://github.com/IgniteUI/igniteui-react-examples/blob/master/samples/maps/geo-map/marker-template/src/MapMarkerTemplate.tsx#L71)